test/sagemaker_tests/mxnet/inference/integration/sagemaker/test_gluonnlp.py

```diff
@@ -27,6 +27,7 @@
 
 
 @pytest.mark.skip_py2_containers
+@pytest.mark.skip_eia_containers
```

Comment:
Since these are both based on "get_closest_marker", please ensure that both these tests succeed on the PR.

Reply:
I have tested it on EIA and non EIA context, both the test passes.